cmd/controlplane/controller.go

```diff
@@ -44,12 +44,13 @@ func newControllerCommand() *cobra.Command {
 			version := versionpkg.GetVersion()
 			shardName := os.GetEnv("SHARD_NAME", "")
 
-			startupLogEntry := log.WithFields(log.Fields{
-				"version": version.Version,
-				"commit":  version.GitCommit,
-			})
+			log := logging.LoggerFromContext(ctx)
```

Comment:
See [this comment](https://github.com/akuity/kargo/pull/1791/files#r1556679011) -- same concern.